src/components/article/viewArticleComponents.js

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import Pagination from 'rc-pagination';
+import PropTypes from 'prop-types';
+import Pages from 'lodash';
+import './viewArticle.scss';
+import star from '../../../public/assets/images/images/yellowStar.png';
+import blankStar from '../../../public/assets/images/images/blankRateStar.png';
+import userImage from '../../../public/assets/images/images/userIcon.png';
+// import ArticleCard from './articleCard';
```

Comment:
This module dependencie can be eliminated.

Reply:
Fixed